### DISCUSSION TITLE
Muliple text files or one large one?

### DISCUSSION THREAD

**Discussion by VGPS** [imartinez/privateGPT#743] [General]
I'm building out my db with text files. I potentially have around 200+ files. What would the best approach be:
1. ingest them as I get them.
2. ingest one huge file that contains all of them
3. does it matter?
New at this so all input is appreciated!

**Comment by zxjason** [comment]
I tested it, doesn't matter it is bigger or small. I crushed sometimes, but I think it is because of the memory.

**Comment by VGPS** [comment]
Ok thanks. I'll build it out a few files at a time.
